extensions/adobe/experience/adcloud/search/admeasuresearch-adcloud.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/adcloud/search/admeasuresearch-adcloud",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Ad Cloud Search Aggregated Measure",
```

Comment:
Why benefits does this empty class bring you over using ExperienceEvent?

Reply:
The sole purpose of having this class is to get data based partitioning functionality for ingesting daily aggregated measures/metrics.  The schema and underlying data-sets does not classify as an experience event.  The best outcome in this scenario was to create a new class under ad-cloud namespace.